src/components/Property/RaiseWorkOrder/SelectPriority.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types'
+import { Select } from '../../Form'
+import { useState } from 'react'
+import WarningInfoBox from '../../Template/WarningInfoBox'
+
+const SelectPriority = ({
+  priorities,
+  onPrioritySelect,
+  register,
+  errors,
+  priorityCode,
+  priorityCodesWithoutDrs,
+}) => {
+  const [drsScheduled, setDrsScheduled] = useState(false)
+
+  const onSelect = (e) => {
+    setDrsScheduled(priorityCodesWithoutDrs.includes(e.target.value))
+    onPrioritySelect(e)
```

Comment:
Suggest we change this

```
const selectedCode = parseInt(e.target.value)
setDrsScheduled(priorityCodesWithoutDrs.includes(selectedCode))
onPrioritySelect(selectedCode)
```

That last call would work if the parent just passes `setPriorityCode` straight down as the `onPrioritySelect` prop. Then we should be able to delete `onPrioritySelect` in the parent.

Reply:
My earlier code example in Slack while exploring this didn't include integer parsing so that's probably the cause here.